code/API_definitions/qos-profiles.yaml

```diff
@@ -206,6 +206,14 @@ components:
           example: "QoS profile for video streaming"
         status:
           $ref: "#/components/schemas/QosProfileStatusEnum"
+        countryAvailability:
+          description: |
+            A list of countries, and optionally networks, for which the API provider makes the profile available.
+            - When profiles are filtered for a specific device (i.e. using a 3-legged access token or an explicit device filter parameter), the profile can be used by that device in any of these countries when connected to any of the listed networks (assuming roaming is permitted for networks that are not the device's home network), or the API provider's own network if no networks are listed
```

Comment:
I try to remember if we have discussed that already, but shouldn't the API request return only profiles which can be actually used by the device within the network where it is currently connected to? Taking for example the available network technologies into consideration)?

Reply:
In other words: if asked for a specific device it might be better not to provide the field at all.